internal/commands/build.go

```diff
@@ -89,21 +89,11 @@ func Build(logger logging.Logger, cfg config.Config, packClient PackClient) *cob
 				}
 			}
 
-			var cfgTrustedBuilder bool
-			for _, trustedBuilder := range cfg.TrustedBuilders {
-				logger.Debugf("Builder %s is trusted", style.Symbol(trustedBuilder.Name))
-				if flags.Builder == trustedBuilder.Name {
-					cfgTrustedBuilder = true
-					break
-				}
-			}
-
-			var suggestedBuilder bool
-			for _, builder := range suggestedBuilders {
-				if flags.Builder == builder.Image {
-					suggestedBuilder = true
-					break
-				}
+			trustBuilder := isTrustedBuilder(cfg, flags.Builder) || flags.TrustBuilder
+			if trustBuilder {
+				logger.Debugf("Builder %s is trusted", style.Symbol(flags.Builder))
+			} else {
+				logger.Warnf("Builder %s is untrusted", style.Symbol(flags.Builder))
```

Comment:
Should this be changed to a `Debugf`?

Reply:
👍 

Yeah, I think so. I don't think this should be something alarming since we do all the right things to make it none concerning. It's really just something we want them to know (in debug).

Also, might be worth adding a few more sentences about the side effects. eg. "executing sensitive phases separate containers." and/or "using a known lifecycle image to execute sensitive phases."

Reply:
Added, let me know what you think.